graphics/opencv4/Portfile

```diff
@@ -19,28 +18,109 @@ maintainers         \
 
 description         Open Source Computer Vision Library
 
-long_description    OpenCV (Open Source Computer Vision Library) is an\
-                    open source computer vision and machine learning\
-                    software library. OpenCV was built to provide a\
-                    common infrastructure for computer vision\
-                    applications and to accelerate the use of machine\
-                    perception in the commercial products. Being a\
-                    BSD-licensed product, OpenCV makes it easy for\
+long_description    OpenCV (Open Source Computer Vision Library) is an \
+                    open source computer vision and machine learning \
+                    software library. OpenCV was built to provide a \
+                    common infrastructure for computer vision \
+                    applications and to accelerate the use of machine \
+                    perception in the commercial products. Being a \
+                    BSD-licensed product, OpenCV makes it easy for \
                     businesses to utilize and modify the code.
 
-homepage            https://opencv.org
+homepage            https://opencv.org/
 
-master_sites        ${github.master_sites}:${github.project}
+if {${name} eq ${subport}} {
```

Comment:
Given how much things have moved around, I'll need to spend some quality time analyzing the full extent of these changes. (Not a criticism whatsoever, simply a heads-up that I need to scrutinize things closely.)

Reply:
I replaced the `python*` variants as `py*-opencv` subports to allow Portfile dependencies.

The latest Portfile uses all your basic code, but as subports now.